### ISSUE TITLE
[Accessibility]A11y_VSCode_KeyboardShortcuts_Keyboard: Transform to UpperCase and Transform to LowerCase keyboard command is not working.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52669]
**Environment Details:** 
VSCode Version : 1.24.0

**Additional Details:**
MAS Violated: MAS2.1.1

**Repro Steps:**
1. Launch VS Code.
2. Open Keyboard Shortcuts.

**Actual:**
Transform to UpperCase(Ctrl+K Ctrl+U) and Transform to LowerCase(Ctrl+K Ctrl+L) keyboard command is not working.

**Expected:**
Both the commands should work for converting the texts into lowercase and uppercase.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to convert the cases of the characters.

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for Reference: 
![uppercase and lowercase](https://user-images.githubusercontent.com/39204909/41893746-6cea28ba-793a-11e8-96ca-f13016e9715f.JPG)

[A11y VSCode KeyBoardShortcuts UpperCase LowerCase.pptx](https://github.com/Microsoft/vscode/files/2131981/A11y.VSCode.KeyBoardShortcuts.UpperCase.LowerCase.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
Could this be #52681? There is no screencast attached.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Video is already there in "Attachment for Reference" section, also attached the screenshot. Please refer the video for more clarification.

**Comment by isidorn** [comment]
Can not reproduce, duplicate, and these keybinding are not the default in latest insders
